src/request-data/user-preferences/normalizer.js

```diff
@@ -0,0 +1,36 @@
+const VUE_PROPERTY_PREFIX = '__v_'; // Empirically established. I couldn't find documentation on it.
```

Comment:
@brontolosone and I have discussed that it'd be ideal to remove reference to this prefix. That's not something we need to do right away, but I've filed getodk/central-frontend#1046 to track that to-do.